### DISCUSSION TITLE
Mod trying to load before VR starts?

### DISCUSSION THREAD

**Discussion by wugg13** [Okabintaro/SubmersedVR#158] [Q&A]
Hello I have installed everything manually and it appears in the output log that the mod is trying to run before VR has initialized? I'm not exactly sure what's going wrong could someone help? This is one a Quest 3 using airlink on steam. Thank you!
[LogOutput.log](https://github.com/user-attachments/files/18054128/LogOutput.log)

**Comment by Okabintaro** [comment]
Heya, are you using any Beta Branch of Subnautica?
I observed your behaviour on the experimental branch. The legacy one won't work either.
Just yesterday I was able to play the game with the same setup, see: https://github.com/Okabintaro/SubmersedVR?tab=readme-ov-file#the-game-doesnt-work-what-could-be-wrong
